crates/libs/sys/src/Windows/Win32/Networking/HttpServer/mod.rs

```diff
@@ -1043,7 +1040,7 @@ pub const HTTP_REQUEST_AUTH_FLAG_TOKEN_FOR_CACHED_CRED: u32 = 1u32;
 #[cfg(feature = "Win32_Foundation")]
 pub struct HTTP_REQUEST_AUTH_INFO {
     pub AuthStatus: HTTP_AUTH_STATUS,
-    pub SecStatus: i32,
+    pub SecStatus: ::windows_sys::core::HRESULT,
```

Comment:
SECURITY_STATUS is always an HRESULT?

Reply:
It seems to be, at least in some compilation modes, just a raw status code.

Reply:
Additional context @ https://github.com/microsoft/win32metadata/issues/603

It appears `SECURITY_STATUS` grew up from its 16-bit Windows days and is now just a HRESULT. Metadata already mapped SEC_xx_xx constants to HRESULTs; I simply finished the job, so to speak, by making the two types equivalent so we get richer code gen. I spot checked a few API impls (ncrypt, sspi, etc.), particularly `AcceptSecurityContext`, `SslOpenPrivateKey`, and `NCryptGetProperty`, and they all appear to return HRESULTs.

If you're seeing otherwise, holler and we'll get that change reverted.